spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -1447,6 +1454,23 @@ public void setRandomBackOff(Boolean randomBackOff) {
 				this.randomBackOff = randomBackOff;
 			}
 
+			private Topic validate() {
```

Comment:
I won't mind, but I don't know Spring Boot development process in details, so I'll defer a decision for a proper validation and its place to respective Spring Boot team members.
Thanks

Reply:
Sure, thanks. Just for reference, it seems we already have some validation in this `buildProperties` method of the `KafkaProperties` class, so I thought it seemed ok there. Of course, I'm ok with validating any other way or place.

https://github.com/spring-projects/spring-boot/blob/c58e24abd9a415f7811dfc7bc3730087df23dd15/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java#L1216-L1218

Thanks for looking into this.